examples/utils/bundle-size-tests/src/test/checkSizes.spec.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import { readFileSync } from "node:fs";
+
+// Since bundle size reporter is broken, and doesn't block merges, do a sanity check here
```

Comment:
Do we have work tracking fixing the reporter? Should we leave a TODO to come back and do something different once that's been fixed?

Reply:
Abram has an item that we are waiting on some external team to resolve. You can check in with him. Regardless even if the bot worked, it just reports changes and doesn't block anything so we would still want this comment, minus the not about it being broken.

Reply:
```suggestion
// Since bundle size analysis doesn't block regressions, do a sanity check here.
// This bundle should remain its current tiny size for the foreseeable future so putting a hard limit on its size should be ok.
// Additionally, this specific bundling scenario is regressed in the past, so protecting it with a regression test is known to have some value.
```